samples/app/src/main/java/com/pingidentity/samples/app/env/EnvViewModel.kt

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) 2024 Ping Identity. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package com.pingidentity.samples.app.env
+
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.datastore.preferences.core.edit
+import androidx.datastore.preferences.core.stringPreferencesKey
+import androidx.lifecycle.ViewModel
+import com.pingidentity.android.ContextProvider.context
+import com.pingidentity.davinci.DaVinci
+import com.pingidentity.davinci.module.Oidc
+import com.pingidentity.davinci.plugin.DaVinci
+import com.pingidentity.logger.Logger
+import com.pingidentity.logger.STANDARD
+import com.pingidentity.oidc.OidcClient
+import com.pingidentity.oidc.OidcClientConfig
+import com.pingidentity.samples.app.User
+import com.pingidentity.samples.app.settingDataStore
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.launch
+
+val test by lazy {
```

Comment:
Can we remove those environments? We should avoid expose the details of the test environments.

Reply:
Updated.